### ISSUE TITLE
Simplify Data URLs

### ISSUE THREAD

**Issue by jakevdp** [ellisonbg/altair#111]
To create a visualization currently from a URL we use:
```python
Chart(Data(url='http://some.url/some/path.json', format='json')).mark_point()
```
I think we could pretty easily shorten this to
```python
Chart('http://some.url/some/path.json').mark_point()
```
without introducing any ambiguity.

**Comment by ellisonbg** [comment]
+1

On Mon, May 23, 2016 at 7:06 PM, Jake Vanderplas <notifications@github.com>
wrote:

> To create a visualization currently from a URL we use:
>
> Chart(Data(url='http://some.url/some/path.json', format='json')).mark_point()
>
> I think we could pretty easily shorten this to
>
> Chart('http://some.url/some/path.json').mark_point()
>
> without introducing any ambiguity.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ellisonbg/altair/issues/111>
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by jakevdp** [comment]
Done in https://github.com/ellisonbg/altair/commit/81eb6caa9a3874408be45d6371579899e4ae08bf
